### ISSUE TITLE
error when opening editor for attached sources

### ISSUE THREAD

**Issue by gayanper** [ballerina-platform/lsp4intellij#68]
```
java.lang.NullPointerException
	at java.util.concurrent.ConcurrentHashMap.get(ConcurrentHashMap.java:936)
	at java.util.concurrent.ConcurrentHashMap.containsKey(ConcurrentHashMap.java:964)
	at org.wso2.lsp4intellij.client.languageserver.wrapper.LanguageServerWrapper.connect(LanguageServerWrapper.java:278)
	at org.wso2.lsp4intellij.IntellijLanguageClient.lambda$editorOpened$0(IntellijLanguageClient.java:196)
	at com.intellij.openapi.application.impl.ApplicationImpl$1.run(ApplicationImpl.java:311)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
```
This happens when opening a attached source file, for example if you have a library with source jar attached and if you open that file the above error is thrown.

**Comment by NipunaRanasinghe** [comment]
Closing since fixed by #74. @gayanper Really appreciate your contributions towards the plugin stability

**Comment by gayanper** [comment]
@NipunaRanasinghe I think this is a great plugin, i see a future where we can add more language support to IDEA, so it will be my pleasure to help you guys with what i can to take this forward.
